doc/source/notebooks/tips_and_tricks.ipynb

```diff
@@ -418,7 +418,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Or you can `anchor()` your model to the session after the optimisation step. After that your parameters' cache have been updated."
+    "Or you can `anchor()` your model to the session after the optimisation step. The `anchor()` updates parameters' cache. **The `anchor()` is significantly more time consuming than `read_value()`, do not call it too often without necessity.**"
```

Comment:
is anchor(session) also slower than read_value(session)? or does it just seem like that?

Reply:
```suggestion
    "Or you can `anchor(...)` your model to the session after the optimisation step. The `anchor()` updates parameters' cache. **The `anchor(...)` is significantly more time consuming than `read_value(...)`, do not call it too often without necessity.**"
```

Reply:
```suggestion
    "Or you can `anchor(session)` your model to the session after the optimisation step. The `anchor()` updates parameters' cache. **The `anchor(session)` is significantly more time consuming than `read_value(session)`, do not call it too often without necessity.**"
```